debian/pop-default-settings.postinst

```diff
@@ -84,12 +93,10 @@ case "$1" in
         done
 
         # Add Pop proprietary repo if it is missing.
-        RELEASE="$(lsb_release -cs)"
-        REPO="deb http://apt.pop-os.org/proprietary ${RELEASE} main"
-        if ! grep "${REPO}" /etc/apt/sources.list > /dev/null
+        if ! grep -r "${REPO}" /etc/apt/ > /dev/null
```

Comment:
It would probably be better to use the following as a test:

```
if ! apt-cache policy | grep "${REPO}" > /dev/null
```

This would check if the repo is currently configured, ignoring `~` and `.save` files and the like

Reply:
That's a good idea! I'll set that up.

Reply:
Er, wait. Actually, I think we don't want that because it would re-add it if the user decided to disable it.

Reply:
From slack:

```
shopt -s nullglob
if ! grep "${REPO}" /etc/apt/sources.list{,.d/*.list,.d/*.source} > /dev/null
```

Reply:
Yep, just caught that